README.md

```diff
@@ -61,6 +61,19 @@ MY_WEIRD_ENV_VAR=123_ABC #required,format=\d{3}_\w{3}
 
 In the above example, `\d{3}_\w{3}` is converted to a regexp and the value is checked against it.
 
+## If you use Docker, read this
+
+Docker automatically reads `.env` files present in the project's root. What this means is that, if you use `dotenv_validator` in an app you run in a docker container, you might get exceptions or warnings about your variables being in the wrong format even though they're right. The reason is that, when running `docker-compose up`, docker-compose parses the `.env` file before the Rails application starts. It reads each line as is with a really simple parser (no quotes, comments and trailing spaces handling).
```

Comment:
I wouldn't say `Docker reads ...` here, it's not Docker, but Docker Compose, and only when you run `docker-compose up` according to the docs. Since people can use docker without docker compose, and dunning `docker-compose run app /bin/bash` for example won't have this issue

Maybe say `If you use Docker Compose` in the title, and update the first sentence with something like `Docker Compose automatically reads `.env` files present in the project's root when running docker-compose up.`?

Reply:
How about this?

```suggestion
## If you use `docker-compose`, read this

Docker Compose automatically reads `.env` files present in the project's root when running `docker-compose up`. What this means is that, if you use `dotenv_validator` in an app you run using Docker Compose, you might get exceptions or warnings about your variables being in the wrong format even though they're right. The reason is that, when running `docker-compose up`, docker-compose parses the `.env` file before the Rails application starts. It reads each line as is with a really simple parser (no quotes, comments and trailing spaces handling).
```